server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -475,6 +485,14 @@ private ServletHolder getRestServlet() {
     return new ServletHolder(new ServletContainer(rc));
   }
 
+  private ServletHolder getRestV2Servlet() {
+    final ResourceConfig rc = new ResourceConfig().packages("org.apache.accumulo.monitor.next")
```

Comment:
It looks like this servlet is being configured identically to the previous `/rest` servlet. If the classes are moved into the `.rest` package, then there's no need to set up a second servlet at all.